src/Chamilo/CoreBundle/Entity/TrackELogin.php

```diff
@@ -18,9 +18,14 @@ class TrackELogin
     /**
      * @var int
```

Comment:
Just as a reminder, this "@ orm" changes mean changes in the database. So they are not allowed in 1.11.x.
You can remove the "@" and then doctrine will ignore those changes, then in configuration.dist you can add a note to add the "@".